docs/release-process.md

```diff
@@ -0,0 +1,173 @@
+
+### Left-behind release branches
+
+When performing the "Create a release pull request" step described above, release-plz sometimes updates the existing release branch and sometimes creates a new release branch with a new pull request. We don't understand how it makes that choice. When it creates a new pull request, it closes the old pull request, but does not delete the old release branch.
+
+In order to reduce the signal noise in the git repo, we've added a step in the release process to search for and delete old release branches. Search for "Clean up stale release-plz branches" in the [`release.yml`](../.github/workflows/release.yml) task.
+
+### `c2pa` crate accidentally published a 1.0.0 release
+
+When building and testing an earlier version of our release tooling (before adopting release-plz), we accidentally published a [version 1.0.0 of the `c2pa` crate](https://crates.io/crates/c2pa/1.0.0). We immediately yanked that version, but that means version 1.0.0 is permanently unavailable. When we do decide to release 1.0 of the `c2pa` crate, we will have to manually bump the version number to some other version (probably 1.0.1).
+
+## Troubleshooting
+
+### How to recover if the publish step fails or partially fails
+
+We have safeguards in place to ensure that a release PR will result in a successful publish, but they are not 100% effective. When there is a failure in the workflow, it's important to keep the core mental model in mind. (See "how it works" earlier in this document.) The following strategy _should_ work when release-plz fails to publish one or more crates from this project, though it may need some adaptation based on the specific failure mode:
+
+* **Read the logs to understand why the release did not go as planned.** You'll find this in the ["Release-plz" section of the Actions tab](https://github.com/contentauth/c2pa-rs/actions/workflows/release.yml) for this project. As one example, consider [this recent failure](https://github.com/contentauth/c2pa-rs/actions/runs/14339524696/job/40212226853), in which the crate failed to compile during the publish step. (Be aware that `cargo publish` uses a subtly different compilation environment than typical builds, which appears to be the root cause for this failure.)
+
+* **Resolve the issues indicated by the log.** This will vary depending on the failure mode.
+
+* **_ONLY_ for crates that failed to publish, manually revert `Cargo.toml` and `CHANGELOG.md` files.** Remember that release-plz will only generate a new release PR for crates where the `Cargo.toml` version exactly matches what's published on [crates.io](https://crates.io). Be sure to delete the generated section in `CHANGELOG.md` that failed to publish; otherwise, release-plz will raise an error when trying to generate the next release PR.
```

Comment:
Can you clarify what branch of what we are supposed to revert here? Is it main, the PR or the release PR that was generated?

Reply:
Updated the text.